presto-tests/src/test/java/com/facebook/presto/tests/tpch/TpchQueryRunner.java

```diff
@@ -44,6 +44,21 @@ public static DistributedQueryRunner createQueryRunner(Map<String, String> extra
     public static DistributedQueryRunner createQueryRunner(Map<String, String> extraProperties, Map<String, String> coordinatorProperties)
             throws Exception
     {
+        DistributedQueryRunner queryRunner = createQueryRunnerWithoutCatalogs(extraProperties, coordinatorProperties);
+        try {
+            queryRunner.createCatalog("tpch", "tpch");
+
+            return queryRunner;
+        }
+        catch (Exception e) {
+            queryRunner.close();
+            throw e;
+        }
+    }
+
+    public static DistributedQueryRunner createQueryRunnerWithoutCatalogs(Map<String, String> extraProperties, Map<String, String> coordinatorProperties)
```

Comment:
does this method makes sense anymore if it doesn't `createCatalog`? Who is the caller of this method? Make it private maybe.